docs/src/extend/shareable-configs.md

```diff
@@ -8,17 +8,23 @@ eleventyNavigation:
 
 ---
 
-The configuration that you have in your `.eslintrc` file is an important part of your project, and as such, you may want to share it with other projects or people. Shareable configs allow you to publish your configuration settings on [npm](https://www.npmjs.com/) and have others download and use it in their ESLint projects.
+To share your `.eslintrc` file, create a **shareable config**. You can publish your shareable config on [npm](https://www.npmjs.com/) so that others can download and use it in their ESLint projects.
```

Comment:
Because we will be renaming the config file, let's go generic here.
```suggestion
To share your configuration file, create a **shareable config**. You can publish your shareable config on [npm](https://www.npmjs.com/) so that others can download and use it in their ESLint projects.
```

Reply:
I think we should avoid "file" because shareable configs cannot be used with [`--config`](https://eslint.org/docs/latest/use/command-line-interface#-c---config) and this is rather sharing a configuration object than sharing a file.

 ```suggestion
To share your ESLint configuration, create a **shareable config**. You can publish your shareable config on [npm](https://www.npmjs.com/) so that others can download and use it in their ESLint projects.
```

Thoughts?

Reply:
Makes sense to me.